projects/storefrontlib/cms-structure/routing/default-routing-config.ts

```diff
@@ -4,75 +4,81 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { RoutesConfig, RoutingConfig } from '@spartacus/core';
+import { FeatureToggles, RoutingConfig } from '@spartacus/core';
+import { inject } from '@angular/core';
 
-export const defaultStorefrontRoutesConfig: RoutesConfig = {
-  home: { paths: [''] },
-  notFound: { paths: ['not-found'] },
+export const defaultRoutesConfigFactory: () => RoutingConfig = () => {
```

Comment:
if we have agreed that moving to factory is not breaking change, i'm happy with it.